nrf802154_sniffer/nrf802154_sniffer.lua

```diff
@@ -0,0 +1,19 @@
+nrf802154_sniffer_proto = Proto("nrf802154_sniffer","nRF 802.15.4 Sniffer Data")
+nrf802154_sniffer_proto.fields.channel = ProtoField.uint16("nrf802154_sniffer.channel", "Channel", base.DEC)
+nrf802154_sniffer_proto.fields.rssi = ProtoField.int16("nrf802154_sniffer.rssi", "RSSI", base.DEC)
+nrf802154_sniffer_proto.fields.lqi = ProtoField.uint16("nrf802154_sniffer.lqi", "LQI", base.DEC)
+
+-- create a function to dissect it
+function nrf802154_sniffer_proto.dissector(buffer, pinfo, tree)
+    HEADER_LEN = 6
+    if buffer:len() > HEADER_LEN then
+        local t = tree:add(nrf802154_sniffer_proto, buffer(), "nRF 802.15.4 Sniffer Data")
```

Comment:
You may want to use ```buffer(0,HEADER_LEN)``` here to show only 6 bytes in the bytes view for the "nRF 802.15.4 Sniffer Data" tree entry.